src/test/scala/is/hail/variant/CallSuite.scala

```diff
@@ -0,0 +1,17 @@
+package is.hail.variant
+
+import org.scalatest.testng.TestNGSuite
+import org.testng.annotations.Test
+
+class CallSuite extends TestNGSuite {
```

Comment:
Also test read/write with a Call object.  You can achieve this by adding TCall to the Type.genSized type generator (so you don't need an explicit test here).  I think it would fail right now because Call objects can not be written as Parquet.  Making them Ints will fix that.  The other option would be to convert between Call and Int in SparkAnnotationImpex, but that would be very expensive.

I'd also add some basic tests in ExprSuite following the genotype tests.  (Search for Genotype, isHomRef, etc.)